README.md

```diff
@@ -1,2 +1,178 @@
-# ReportPortalIntegration
-Utility to easily integrate ReportPortal with your automation framework
+# Introduction
+
+1. Utility to easily integrate ReportPortal with your JAVA-TestNG automation framework
+2. This is a library which implements basic utilities
+   for [ReportPortal](https://reportportal.io/docs/) `(com.epam.reportportal)`
+
+# Utilities
+
+1. `ReportPortalLogger`: For logging messages to ReportPortal at different levels like DEBUG, INFO WARN
+2. `ScreenShotManager`: Takes care of following
```

Comment:
do we need ScreenShotManager and CommandLineExecutor as public methods? We can just have the ReportPortalLogger methods public, and everything else private/package-protected, right?

Reply:
I explained the reasons above. Let me know your thoughts?